examples/cli/src/test/java/io/unitycatalog/cli/table/CliExternalTableCreationTest.java

```diff
@@ -0,0 +1,155 @@
+package io.unitycatalog.cli.table;
+
+import io.unitycatalog.cli.catalog.CliCatalogOperations;
+import io.unitycatalog.cli.delta.DeltaKernelUtils;
+import io.unitycatalog.cli.schema.CliSchemaOperations;
+import io.unitycatalog.client.ApiException;
+import io.unitycatalog.client.model.*;
+import io.unitycatalog.server.base.BaseServerTest;
+import io.unitycatalog.server.base.catalog.CatalogOperations;
+import io.unitycatalog.server.base.schema.SchemaOperations;
+import io.unitycatalog.server.base.table.TableOperations;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+
+import java.io.IOException;
+import java.nio.file.*;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.List;
+import java.util.UUID;
+
+import static io.unitycatalog.server.utils.TestUtils.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class CliExternalTableCreationTest extends BaseServerTest {
+
+
+    private CatalogOperations catalogOperations;
+    private SchemaOperations schemaOperations;
+    private TableOperations tableOperations;
+
+    private static final String SCHEMA_FULL_NAME = CATALOG_NAME + "." + SCHEMA_NAME;
+    private static final String TABLE_FULL_NAME = CATALOG_NAME + "." + SCHEMA_NAME + "." + TABLE_NAME;
+
+    @Before
+    public void setUp() {
+        super.setUp();
+        catalogOperations = new CliCatalogOperations(serverConfig);
+        schemaOperations = new CliSchemaOperations(serverConfig);
+        tableOperations = new CliTableOperations(serverConfig);
+        try {
+            catalogOperations.createCatalog(CATALOG_NAME, "Test catalog");
+            schemaOperations.createSchema(new CreateSchema().name(SCHEMA_NAME).catalogName(CATALOG_NAME).comment("Test schema"));
+        } catch (Exception e) {
+            // Ignore
```

Comment:
Shouldn't the test fail if create catalog / create schema fail?

Reply:
Added the fail clause in catch.